master_preset.xml

```diff
@@ -1545,7 +1545,7 @@
     </chunk>
     <chunk id="conditional_fee">
         <combo key="fee:conditional" text="Fee @" match="none">
-            <list_entry value="yes @ Mo-Fr 09:00-19:00"/> 
+            <list_entry value="yes @ Mo-Fr 09:00-19:00"/>
             <list_entry value="no @ stay &lt; 1 hour"/>
         </combo>
```

Comment:
Does this and the following change have anything to do with adding sauna type? Just checking.

Reply:
@simonpoole They are just cosmetic changes that were automatically done by my IDE while adding the sauna type.